apps/web/src/pages/templates/editor_v2/TemplateDetailsPageV2.tsx

```diff
@@ -0,0 +1,70 @@
+import { Button, IconButton } from '@novu/novui';
+import { css } from '@novu/novui/css';
+import { IconCable, IconPlayArrow, IconSettings } from '@novu/novui/icons';
+import { useNavigate, useParams } from 'react-router-dom';
+import { WorkflowsPageTemplate } from '../../../studio/components/workflows/layout';
+import { useTemplateController } from '../components/useTemplateController';
+import { parseUrl } from '../../../utils/routeUtils';
+import { ROUTES } from '../../../constants/routes';
+import { WorkflowFloatingMenu } from '../../../studio/components/workflows/node-view/WorkflowFloatingMenu';
+import { WorkflowNodes } from '../../../studio/components/workflows/node-view/WorkflowNodes';
+import { WorkflowBackgroundWrapper } from '../../../studio/components/workflows/node-view/WorkflowBackgroundWrapper';
+
+export const TemplateDetailsPageV2 = () => {
+  const { templateId = '' } = useParams<{ templateId: string }>();
+
+  const { template: workflow } = useTemplateController(templateId);
+
+  const title = workflow?.name || '';
+  const navigate = useNavigate();
+
+  const handleSettingsClick = () => {};
+  const handleTestClick = () => {
+    // navigate(ROUTES.STUDIO_FLOWS_TEST);
+  };
+
+  return (
```

Comment:
Been thinking here of what we should reuse and what should stay at this level here. I think we should still allow control for the host page for some layout concerns as they might evolve differently in the future.